report.md

```diff
@@ -64,10 +64,52 @@
 ## Test
 
 ### Test Plan
-[Describe the testing approach and methodologies.]
+This test plan descripes our current idea of the approach, tools and strategies we are going to be using to ensure the quality and stability of the full-stack application developed using the specified tech stack in this repport for the front and backend. 
+#### Testing opbjectives
+- Ensure application functions correctly across all integrated systems (backend, frontend, database)
+- Validate application security features particularly token mangement and authorization (OIDC)
```

Comment:
mangement->management

Reply:
Also, we do not use OIDC anyways, see https://github.com/Complex-systems-and-devops-gruppe-8/backend/tree/649981b1627ea387cecb2abfd4cf2097307b487c/rest-client-quickstart/src/main/java/org/csdg8/auth